dashboard/app/models/user.rb

```diff
@@ -113,7 +113,7 @@ class User < ActiveRecord::Base
   # Include default devise modules. Others available are:
   # :token_authenticatable, :confirmable,
```

Comment:
Note that `:confirmable` already appears in the list of (unused) available devise modules.